pages/productListing.html

```diff
@@ -0,0 +1,330 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://splash-ui.vercel.app/CSS/main.css">
+    <link rel="stylesheet" href="/css/productListing.css">
+    <title>Sneaker Street</title>
+</head>
+<body>
+    <header class="header-main">
+        <nav class="nav nav-dark" style="position: relative;">
+            <h3>SNEAKER STREET</h3>
+            <button class="hamburger">
+                <i class="fa fa-bars fa-2x" aria-hidden="true"></i>
+            </button>
+            <div class="search-container">
+                <input type="text" placeholder="🔍  Search.." name="search">
+            </div>
+            <ul class="nav-list">
+                <li><button class="btn btn-secondary">Login</button></li>
+                <li><button class="btn btn-icon">
+                    <i class="fa fa-heart fa-lg" aria-hidden="true"></i>
+                </button></li>
+                <li><button class="btn btn-icon">
+                    <i class="fa fa-shopping-cart fa-lg" aria-hidden="true"></i>
+                </button></li>
+            </ul>
+        </nav>
+    </header>
+
+    <div class="content-wrapper">
+        <div class="filter-wrapper">
+            <header class="filter-header">
+                <h3>Filters</h3>
+                <button class="btn-filter">Clear</button>
+            </header>
+            <section class="filter-section">
+                <h4>Price</h4>
+                <input type="range" min="500" max="5500" step="1000" list="pricTick">
+                <datalist id="pricTick">
+                    <option label="500" value="500"></option>
+                    <option label="500" value="1500"></option>
+                    <option label="500" value="2500"></option>
+                    <option label="500" value="3500"></option>
+                    <option label="500" value="4500"></option>
+                </datalist>
+            </section>
+            <section class="filter-section">
+                <h4>Gender</h4>
+                <ul class="filter-list">
+                    <li>
+                        <label class="filter-label">
+                            <input type="radio" name="allgender" value="allgender" class="filter-input">
+                            All Genders
+                        </label>
+                    </li>
+                    <li>
+                        <label class="filter-label">
+                            <input type="radio" name="women" value="women" class="filter-input">
+                            Women
+                        </label>
+                    </li>
+                    <li>
+                        <label class="filter-label">
+                            <input type="radio" name="men" value="men" class="filter-input">
+                            Men
```

Comment:
Radio group should share the same name so that only one can be selected,
currently all are being selected.